src/components/Card/index.examples.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import {storiesOf} from '@kadira/storybook'
+import Card from '.'
+
+storiesOf('Static: Cards')
+  .addWithInfo('Course Card', () => (
+    <div className='flex justify-center items-center bg-dark-navy pv5'>
```

Comment:
Since you're using the same class names in two places, you could make a variable for it:

`const dispClasses = 'flex justify-center items-center bg-dark-navy pv5'`

then `<div className={divClasses}>`